[2025-04-10T11:42:13.501+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-10T11:42:13.512+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-04-09T00:00:00+00:00 [queued]>
[2025-04-10T11:42:13.517+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-04-09T00:00:00+00:00 [queued]>
[2025-04-10T11:42:13.518+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-10T11:42:13.524+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): meteo_operations.collect_meteo_data> on 2025-04-09 00:00:00+00:00
[2025-04-10T11:42:13.529+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'meteo_operations.collect_meteo_data', 'scheduled__2025-04-09T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp0wfpsg49']
[2025-04-10T11:42:13.531+0000] {standard_task_runner.py:105} INFO - Job 2: Subtask meteo_operations.collect_meteo_data
[2025-04-10T11:42:13.532+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=188) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-10T11:42:13.533+0000] {standard_task_runner.py:72} INFO - Started process 210 to run task
[2025-04-10T11:42:13.562+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-04-09T00:00:00+00:00 [running]> on host 9a7b018a2323
[2025-04-10T11:42:13.611+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='meteo_operations.collect_meteo_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-09T00:00:00+00:00'
[2025-04-10T11:42:13.613+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-10T11:42:14.399+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 21.4901, 'longitude': 39.1862, 'timestamp': '2025-04-10 00:00:00', 'temperature': 25.823501586914062, 'humidity': 77.0, 'wind_speed': 18.861387252807617, 'pm2_5': 243.10000610351562, 'co': 174.0}
[2025-04-10T11:42:14.425+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-10T11:42:14.426+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=meteo_operations.collect_meteo_data, run_id=scheduled__2025-04-09T00:00:00+00:00, execution_date=20250409T000000, start_date=20250410T114213, end_date=20250410T114214
[2025-04-10T11:42:14.469+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-10T11:42:14.483+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-10T11:42:14.485+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
